.drone.star

```diff
@@ -253,15 +253,15 @@ def docker(config):
                     },
                     {
                         "name": "redis",
-                        "image": "library/redis:4.0",
+                        "image": "docker.io/redis:6",
                     },
                     {
                         "name": "email",
-                        "image": "inbucket/inbucket",
+                        "image": "docker.io/inbucket/inbucket",
                     },
                     {
                         "name": "selenium",
-                        "image": "selenium/standalone-chrome-debug:3.141.59-oxygen",
+                        "image": "docker.io/selenium/standalone-chrome:111.0",
```

Comment:
Note: `selenium/standalone-chrome-debug:3.141.59-oxygen"` gets used in some core CI because of issues with the PHP-behat-mink-selenium test stack that does not support some newer webdriver stuff.

But that is not a problem here, if we are not running those web-brower tests. Let's see...

Reply:
You are right, and I just wanted to test it. For core, we need to take actions as well IMO as the used `selenium/standalone-chrome-debug` container is not maintained anymore, and the latest one is now two years old already.

Reply:
Tests failed https://drone.owncloud.com/owncloud-docker/server/1532/16/12 Will revert it for now, but we still might need to look into a way to use a maintained container for this, what do you think @phil-davis ?

Reply:
Issue https://github.com/owncloud/core/issues/37113

I haven't noticed any fixes/enahncements for the Behat-Mink-selenium stack recently. "someone" needs to look into it again. I will get one of our guys to see if there is any resolution yet...

Reply:
Thanks!